load-test/single_msg_load.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/sh
+
+FROM=$1
+if [ -z $FROM ]
+then
+    FROM=1
```

Comment:
Not sure I understand, is this a bech32 address? What happens when we assign simple integer value here

Reply:
same as above